pkg/comp-functions/functions/vshn-postgres-func/loadBalancer.go

```diff
@@ -27,37 +26,55 @@ func AddLoadBalancerIPToConnectionDetails(ctx context.Context, iof *runtime.Runt
 		return runtime.NewFatalErr(ctx, "Cannot get composite from function io", err)
 	}
 
-	if comp.Spec.Parameters.Network.ServiceType != "LoadBalancer" {
-		return runtime.NewNormal()
+	k8sservice := &v1.Service{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      fmt.Sprintf("%s-%s", comp.GetName(), serviceName),
+			Namespace: getInstanceNamespace(comp),
+		},
+		Spec: v1.ServiceSpec{
+			Selector: map[string]string{
+				"role": "master",
+			},
+			Ports: []v1.ServicePort{
+				{
+					Name:     "pgport",
+					Port:     5432,
+					Protocol: v1.ProtocolTCP,
+					TargetPort: intstr.IntOrString{
+						Type:   intstr.String,
+						StrVal: "pgport",
+					},
+				},
+			},
+		},
 	}
 
-	cluster, err := getSGCluster(ctx, iof)
-	if err != nil {
-		return runtime.NewFatalErr(ctx, "Cannot get cluster from function io", err)
-	}
-	if err := updateClusterForLoadBalancer(cluster, ctx, iof); err != nil {
-		return runtime.NewFatal(ctx, "Cannot put cluster into function io")
+	if comp.Spec.Parameters.Network.ServiceType == "LoadBalancer" {
+		k8sservice.Spec.Type = v1.ServiceTypeLoadBalancer
+	} else {
+		k8sservice.Spec.Type = v1.ServiceTypeClusterIP
 	}
 
-	serviceToApply, err := createServiceForLoadBalancer(comp)
-	if err != nil {
-		return runtime.NewFatalErr(ctx, "Cannot create load balancer service", err)
+	fmt.Println(comp.Spec.Parameters.Network.ServiceType)
```

Comment:
This looks like a debug statement, please remove or use the logger with `V(1)` to make it a proper debug log.

Reply:
whoopsi, leftover from tests, sry